fbchat/models.py

```diff
@@ -216,6 +216,12 @@ def __init__(self, uid=None):
         """Represents a Facebook attachment"""
         self.uid = uid
 
+class DeletedMessage(Attachment):
```

Comment:
How is the `DeletedMessage` event an attachment? Is it attached to the message which is deleted? If so, why not just make a `deleted` attribute on the message?

Reply:
It's weird, but it is an attachment, but I'll try somehow to make `deleted` attribute